offline-runtime/api/src/com/intellij/rt/coverage/offline/api/CoverageRuntime.java

```diff
@@ -43,6 +43,11 @@ public static List<ClassCoverage> collectClassfileData(final List<byte[]> classF
     return CoverageCollector.collect(projectData, classFinder);
   }
 
+  public static void dumpIcReport(DataOutput output, File errorFile) throws IOException {
+    ErrorReporter.setPath(errorFile.getPath());
```

Comment:
This does not work completely as expected. We need to set a logging path in the beginning, alternatively, a predefined log file location will be used. Can we have a separate method for log path setting?

Reply:
Because `ErrorReporter` can be called during the tests run, before the `dumpIcReport` method called

Reply:
Indeed )
I forget that logs are not buffered in memory

Reply:
so yes, we need to add a separate method that will be called when the `kotlinx.kover.offline.runtime.KoverInit` callback is triggered